Source/WebCore/animation/KeyframeEffect.h

```diff
@@ -184,6 +188,11 @@ class KeyframeEffect final : public AnimationEffect, public CSSPropertyBlendingC
 
     WebAnimationType animationType() const { return m_animationType; }
 
+#if ENABLE(THREADED_ANIMATION_RESOLUTION)
+    const AcceleratedEffect* acceleratedRepresentation() const { return m_acceleratedRepresentation.get(); }
```

Comment:
Shouldn't this return an optional?

Reply:
I see you check for null at the calling site. I'm not sure what the convention here is. Return the WeakPtr directly?

Reply:
Returning a pointer allows us to distinguish between the lack of an accelerated representation (`nullptr`) or a valid value.